core/templates/core/home_page.html

```diff
@@ -92,12 +92,12 @@ <h3 class="double-height"><a style="color: {{ article.font_style.text_colour.hex
                     {% else %}
                         <div class="simple-feature">
                             {% if article.editors_pick %}
-                                <a href="{% slugurl 'editors-picks' %}"><div class="article-feature editors-pick"><div>Editor's Pick</div></div></a>
+                                <div class="editors-pick-link"><div class="table"><div class="table-cell"><a href="{% slugurl 'editors-picks' %}" class="article-feature editors-pick">Editor's Pick</a></div></div></div>
                             {% endif %}
                             {% if article == self.most_popular_article %}
-                                <a href="{% slugurl 'most-popular' %}"><div class="article-feature most-popular"><div>Most Popular</div></div></a>
+                                <div class="most-popular"><div class="table"><div class="table-cell"><a href="{% slugurl 'most-popular' %}" class="article-feature ">Most Popular</a></div></div></div>
                             {% endif %}
-                            <h3><a href="{{ article.url }}">{{ article.title }}</a></h3>
+                            <h3 class="{% if article.editors_pick %} editors-pick{% endif %}"><a href="{{ article.url }}">{{ article.title }}</a></h3>
```

Comment:
Hi Caroline,
Yes this makes sense now that I know an item can be tagged with multiple things.